baseapp_core/hashids/strategies/legacy/graphql_resolver.py

```
@@ -0,0 +1,47 @@
+            assert graphene_type == only_type, f"Must receive a {only_type._meta.name} id."
+
+        # We make sure the ObjectType implements the "Node" interface
+        if Node not in graphene_type._meta.interfaces:
+            raise Exception(f'ObjectType "{_type}" does not implement the "{Node}" interface.')
+
+        get_node = getattr(graphene_type, "get_node", None)
+        if get_node:
+            return get_node(info, _id)
+
+    def get_pk_from_global_id(self, global_id):
+        gid_type, gid = from_global_id(global_id)
```

Ruff (0.14.10)

Unpacked variable `gid_type` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)